tvb/simulator/plot/utils/analyzers_utils.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
```

Comment:
This module should not be needed for plot Is it?

Reply:
In this moment, it is used in time_series_plotter.py , class TimeSeriesPlotter, method _plot_ts_spectral_analysis_raster (file that is copied from tvb-multiscale). Probably, we have to investigate in depth if we need or not to have the module analyzers_utils.

Reply:
Ok. Then maybe make a smaller utils.py with only used method(s).
Depending on what will be left, we decide on a name for the new utils.
We could even consider putting the remaining function directly in module time_series_plotter.py (next to the place where it is used)

Reply:
Ok. I created a utils.py with all the used methods from plot utils.